src/sass/table/table.scss

```diff
@@ -164,6 +168,32 @@ $density-relaxed-img-radius: 8px;
     padding-inline-end: 0;
 }
 
+.table--loading-state {
+    overflow: hidden;
+}
+
+.table--loading-state table {
+    opacity: 0.15;
+    user-select: none;
+}
+
+.table--loading-state img,
+.table--loading-state .signal {
+    filter: var(--color-media-disabled-filter);
+}
+
+.table--loading-state a {
+    cursor: default;
+    pointer-events: none;
+    text-decoration: none;
+}
+
+.table--loading-state .progress-bar-expressive {
+    bottom: calc(50%);
+    left: calc(50% - 172px);
+    position: sticky;
```

Comment:
Whats the reason behind using position sticky rather than absolute?

Reply:
It was mainly significant when scrolling was permitted as it would keep the loader in place regardless of scroll, but since we've disabled scrolling since, `absolute` makes more sense. I'll change it.